### ISSUE TITLE
Nav component: Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#978]
The aXe a11y tool is finding issues within the Nav component. The area's flagged appear to be the Current Link styles and the ~~Horizontal Nav styles~~ (see screen caps below).

![image](https://user-images.githubusercontent.com/21063328/49044523-0f60e280-f19c-11e8-93ff-0aec642c20d9.png)

**UPDATE:** After rechecking, the styles for horizontal nav (shown below) currently pass. The example above still fails (unless we choose to make the text bold for this color combination).

![image](https://user-images.githubusercontent.com/21063328/49044532-1982e100-f19c-11e8-8958-f31cab953771.png)

Additional information is available at #620

**Comment by seanforyou23** [comment]
Just double checked and this issue is still valid, it's being flagged in our build audit;

<img width="933" alt="Screen Shot 2019-06-06 at 9 16 20 PM" src="https://user-images.githubusercontent.com/5942899/59075764-66104d00-88a0-11e9-88e7-a7edafcb0f56.png">

**Comment by mcarrano** [comment]
@jgiardino @jessiehuff @mcoker Do you know if this is still an issue?

**Comment by mcoker** [comment]
I think we can close it. I ran axe on the the dark and light theme, horizontal, and tertiary nav pages and it passes. We will also be using new axe tool checks, so we can open a new issue if that flags a problem.
